src/storage/store.js

```diff
@@ -17,6 +17,7 @@ export const SCHEMA = {
       additionalProperties: false,
       properties: {
         display_name: { type: "string", pattern: "^[A-Za-z0-9-]{3,32}$" },
+        avatar: { type: "string" },
```

Comment:
I would call this "avatar_url" -- or we may want to consider future proofing this by storing some other higher-level selection choice. (Probably URL is ok though once we get gltf bundles set up)

Reply:
It's actually the id as defined in the `avatars.json` file I added, so I'll go with avatar_id. 
BTW, is there a reason you went with snake_case instead of camelCase for the schema properties?

Reply:
ah cool, makes sense. wrt snake case i just tend to write persistent data structure schemas that way (avro schema, db tables, etc) not sure if this is un-local-storagey :)

Reply:
It's un-javascripty and inconsistent with all our other naming, but I'm not that strongly against it.

Reply:
I vote we store the thing we end up passing to `src` aka either a url or a selector of a "bundled" `a-progressive-asset`, though I prefer we eventually ditch the later for everything but some sort of preloading avatar.

Reply:
I think if we can avoid a URL for now that is probably an good idea since there's a good chance URLs are gonna change for avatars in the near term. (For example, as we figure out the whole deploy pipeline, domain, GLTF bundle URLs, etc.)

I am not opposed to our long term plan being that your current avatar(s) are lists of URLs though.

Reply:
also, wrt "custom URLs for avatars" I believe we can have it both ways. We can have an `avatar_url` field, which if present, is assumed to be the custom override. (And perhaps eventually this can be the field we use for our 1st party avatars.)

The main thing I think we want to be careful about is burning in URLs to the 1st party avatars, because if we decide to change them we'll end up having to blow away peoples' avatar choices and/or maintain local storage migration scripts :/

Reply:
Yeah, I imagine a lot is going to change with the asset bundles, so I'm not going to attempt anything fancy here. It's not difficult to change anyway.